AEPMessaging/Tests/UnitTests/Messaging+PublicApiTest.swift

```diff
@@ -440,17 +462,17 @@ class MessagingPublicApiTest: XCTestCase {
         wait(for: [expectation], timeout: 1)
     }
 
-    func testSetFeedsHandler_noFeeds() {
+    func testSetPropositionsHandler_noFeeds() {
         // setup
-        let expectation = XCTestExpectation(description: "setFeedsHandler should not be called for no feeds in personalization notification response.")
+        let expectation = XCTestExpectation(description: "setPropositionsHandler should not be called for no feeds in personalization notification response.")
```

Comment:
nit:  1. testcase - `testSetPropositionsHandler_noPropositions` 2. ...not be called for no ~feeds~ propositions...